### PR TITLE
Category Grid and Video card row added

### PR DESCRIPTION
**Pull request by t007rushi**
## Category 
 - with video playing cards in the background on hover
## Video card rows 
 - with transitions
 -  will work on slider in next one 

recording

### CODE REVIEW

File: src/frontend/components/Categories/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export const Categories = () => {
+  
+  return (
+    <div className="category-container">
+      {categoryData.map((item) => {
+        return (
+          <div key={item.categoryName} className="category-wrapper">
+            <img src={item.image} alt="items" className="category" />
+            <video autoPlay={true} loop={true} playsInline={true} muted={true}>
+                <source src={item.video} type="video/mp4"/>
```
Comment (MiheerTamkhane) [inline src/frontend/components/Categories/Categories.jsx:14]: may be you should destructure item here.

**Review by MiheerTamkhane — APPROVED** [review]
Code looks good and readable. 
good to merge.

File: src/frontend/components/Row/row.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  transition: transform 450ms;
+}
+
+.title_poster:hover {
+  transform: scale(1.06);
+}
+
+.sliderarrow {
+  font-size: 3rem;
+  width: 50px;
```
Comment (Kothalkarpriya) [inline src/frontend/components/Row/row.css:43]: For more than 10px you can use rem

**Review by Kothalkarpriya — APPROVED** [review]
Code is clean and good, added once suggestion

**Pull request merged**